pkgs/applications/misc/gnome-epub-thumbnailer/default.nix

```
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchurl
+, wrapGAppsNoGuiHook
+, meson
+, ninja
+, pkg-config
+, gnome
+, gdk-pixbuf
+, libarchive
+, libxml2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-epub-thumbnailer";
+  version = "1.7";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/gnome-epub-thumbnailer/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "sha256-S7Ah++RCgNuY3xTBH6XkMgsWe4GpG9e6WGvqDE+il1I=";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsNoGuiHook
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    gdk-pixbuf
+    libarchive
+    libxml2
+  ];
+
+  passthru = {
+    updateScript = gnome.updateScript {
+      packageName = pname;
+      attrPath = "gnome.${pname}";
+    };
```

The `attrPath` is different now.

```suggestion
    updateScript = gnome.updateScript {
      packageName = "gnome-epub-thumbnailer";
    };
```